### PR TITLE
Windows: Enable tensorflow/contrib in Bazel build

### PR DESCRIPTION
**Pull request by meteorcloudy**
This change requires upgrading Bazel to 0.10.0 because we need `cc_import` rule.

I disabled all failing tests on Windows, I will send issues to their owner later and write some tips about how to reproduce and test them.

@gunan @mrry @martinwicke 

FYI @dslomov @laszlocsomor

### CODE REVIEW

File: configure.py [hunk 1]
```diff
@@ -1343,7 +1343,7 @@ def main():
   # environment variables.
   environ_cp = dict(os.environ)
 
-  check_bazel_version('0.5.4')
+  check_bazel_version('0.10.0')
```
Comment (gunan) [inline configure.py]: We should fix the build issues with 0.10 in old branches and then make this change.
We will probably take care of all the cherrypicks later this week.

Comment (gunan) [reply configure.py]: We also need to bump the bazel version on all build infra and all our docker images before we do this.

Comment (meteorcloudy) [reply configure.py]: Sure, we can wait for them.

**Comment by martinwicke** [comment]
What's the status of this? Still WIP for the infra?

**Comment by laszlocsomor** [comment]
@martinwicke , @gunan : FYI @meteorcloudy is on vacation this week and the next, so he's unlikely to give an update.

**Comment by gunan** [comment]
We are working on upgrading the test infra.
I will try to resolve conflicts and rerun tests.

**Review by gunan — APPROVED** [review]

**Comment by martinwicke** [comment]
I'm assuming you're still working on this.

**Comment by meteorcloudy** [comment]
@martinwicke, I just came back from vacation, will continue to work on this today!

**Comment by meteorcloudy** [comment]
Hmm, I'm getting this error?
```
ERROR: T:/src/github/tensorflow/WORKSPACE:41:1: Traceback (most recent call last):
	File "T:/src/github/tensorflow/WORKSPACE", line 41
		tf_workspace()
	File "T:/src/github/tensorflow/tensorflow/workspace.bzl", line 72, in tf_workspace
		check_bazel_version_at_least("0.10.0")
	File "T:/src/github/tensorflow/tensorflow/workspace.bzl", line 62, in check_bazel_version_at_least
		fail("\nCurrent Bazel version is {}, ...))

Current Bazel version is 0.9.0, expected at least 0.10.0
```
Seems like we forget to upgrade Bazel version on Kokoro Windows machines? @yifeif

**Comment by yifeif** [comment]
bazel should be upgraded on windows now

**Comment by meteorcloudy** [comment]
Nice, all tests passed, We can merge this now!

**Comment by meteorcloudy** [comment]
Thanks for the quick fix!
We do have the bazel binary stored on x20.
For 0.11.0, it would be %KOKORO_GFILE_DIR%\bazel-0.11.0-windows-x86_64.exe

**Comment by gunan** [comment]
This is breaking everything internally.
@meteorcloudy I will roll this back now. Please resend it internally.

**Comment by miaout17** [comment]
+1 This broke bazel rules.

**Comment by meteorcloudy** [comment]
@gunan Can you point me to the internal error this change caused? I'll help fixing it.

**Comment by meteorcloudy** [comment]
@miaout17 What do you mean it broke bazel rules? Can you elaborate?

**Comment by gunan** [comment]
@meteorcloudy The issue was during merge. It caused multiple issues during merge, so we had to roll back.

**Pull request merged**